ppstructure/predict_system.py

```diff
@@ -78,6 +80,13 @@ def __init__(self, args):
                     )
                 else:
                     self.table_system = TableSystem(args)
+            if args.formula:
+                args_fomula = deepcopy(args)
```

Comment:
There is a typo in args_fomula which should be args_formula

Reply:
ok, I will modify it